sys/geom/raid/md_sii.c

```diff
@@ -332,6 +332,13 @@ sii_meta_write(struct g_consumer *cp, struct sii_raid_conf *meta)
 
 	pp = cp->provider;
 
+	/* Verify that the sector size is large enough for the metadata */
+	if (pp->sectorsize < sizeof(*meta)) {
+		G_RAID_DEBUG(1, "Sector size %u too small for metadata on %s",
+		    pp->sectorsize, pp->name);
+		return (EINVAL);
+	}
+
```

Comment:
I think this hunk is bogus. We'll have already checked in taste. It can be dropped.

Reply:
Anything else I should know or should do?